models/cart-discount-value-relative/src/generator.ts

```diff
@@ -0,0 +1,18 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TCartDiscountValueRelative } from './types';
+
+// https://docs.commercetools.com/api/projects/cartDiscounts#cartdiscountvaluerelative
+
+const generator = Generator<TCartDiscountValueRelative>({
+  fields: {
+    type: 'relative',
+    permyriad: fake((f) =>
+      f.datatype.number({
+        min: 0,
+        max: 10000,
```

Comment:
It's not explicit in the docs, but:
```Fraction (per ten thousand) the price is reduced by. For example, 1000 will result in a 10% price reduction.```

I imagine 10k is the limit, otherwise customers are...getting more than 100% off 😆